HCaptcha/Classes/HCaptchaWebViewManager.swift

```diff
@@ -12,6 +12,7 @@ internal class HCaptchaWebViewManager: NSObject {
     enum JSCommand: String {
         case execute = "execute();"
         case reset = "reset();"
+        case reload = "reload();"
```

Comment:
Doesn't seems required because there is no `reload` function added to `HCaptcha/Assets/hcaptcha.html`